src/Misc/layoutbin/powershell/Add-AzureGuestAgentCapabilities.ps1

```diff
@@ -0,0 +1,24 @@
+[CmdletBinding()]
+param()
+
+$guestAgentPath = $null
+
+try {
+    $proc = Get-Process 'WindowsAzureGuestAgent'
```

Comment:
@ericsciple thanks

regarding the security context - I verified with the primary scenarios of using Azure VSTS Task Agent extension, it works. And we are fine to not have this captured in other scenarios.

I took the refactoring suggestions. It made the script simpler and cleaner. Thanks.